### PR TITLE
Feature/split nav datagrid from formio datagrid

### PR DESCRIPTION
**Pull request by it-vegard**
Should be reviewed after #178 has been merged to reduce the diff, as this PR builds upon that one.

### CODE REVIEW

File: src/Forms/SummaryPage.jsx [hunk 1]
```diff
@@ -68,19 +69,29 @@ const FormSummaryField = ({ component, value }) => (
   </>
 );
 
-const FormSummaryFieldset = ({ component, submission }) => (
-  <>
-    <dt>{component.legend}</dt>
+const FormSummaryFieldset = ({ component, submission, isDataGridRow }) => (
```
Comment (wen7i) [inline src/Forms/SummaryPage.jsx:72]: Rename ?

File: src/Forms/SummaryPage.jsx [hunk 2]
```diff
@@ -171,4 +181,15 @@ const SummaryContent = styled("main")({
...
   display: "flex",
   flexDirection: "column",
+
+  "& .data-grid__row": {
+    border: `1px solid ${navCssVariabler.navGra60}`,
+    borderRadius: "7px",
+    marginBottom: "1rem",
+    padding: "1.5rem 2rem 0",
+
+    "& .knapp--fjern": {
```
Comment (wen7i) [inline src/Forms/SummaryPage.jsx:191]: Trenger vi dette?

File: src/template/templates/navdesign/field/form.ejs [hunk 3]
```diff
@@ -1,22 +1,22 @@
-{% if (!ctx.label.hidden && ctx.label.labelPosition !== 'bottom' && ctx.component.type !== "radiopanel" && ctx.component.type !== "navCheckbox" && ctx.component.type !== "datagrid" && ctx.component.type !== "navSkjemagruppe") { %}
+{% if (!ctx.label.hidden && ctx.label.labelPosition !== 'bottom' && ctx.component.type !== "radiopanel" && ctx.component.type !== "navCheckbox" && ctx.component.type !== "datagrid" && ctx.component.type !== "navDataGrid" && ctx.component.type !== "navSkjemagruppe") { %}
   {{ ctx.labelMarkup }}
 {% } %}
 
 {% if (ctx.label.hidden && ctx.label.className && ctx.component.validate.required && ctx.component.type !== "radiopanel" && ctx.component.type !== "navCheckbox") { %}
-  <label class="skjemaelement__label{{ctx.component.validate && ctx.component.validate.required ? ' field-required' : ''}}" for="{{ctx.component.key}}" id="{{ctx.component.key}}-label">
+  <label class="{{ctx.label.className}} skjemaelement__label{{ctx.component.validate && ctx.component.validate.required ? ' field-required' : ''}}" for="{{ctx.component.key}}" id="{{ctx.component.key}}-label">
```
Comment (wen7i) [inline src/template/templates/navdesign/field/form.ejs:6]: add styling for hide labels

File: src/template/templates/navdesign/navDataGrid/html.ejs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<table class="table datagrid-table table-bordered
```
Comment (wen7i) [inline src/template/templates/navdesign/navDataGrid/html.ejs:1]: Fungerer det hvis vi fjerner denne filen?

**Review by wen7i — APPROVED** [review]
🥇 🐐 💨

File: src/Forms/SummaryPage.jsx [hunk 5]
```diff
@@ -68,19 +69,37 @@ const FormSummaryField = ({ component, value }) => (
   </>
 );
 
-const FormSummaryFieldset = ({ component, submission }) => (
-  <>
-    <dt>{component.legend}</dt>
+const FormSummaryFieldset = ({ component, submission, isDataGridRow }) => (
```
Comment (rrsslnd) [inline src/Forms/SummaryPage.jsx:72]: Remove?

**Review by rrsslnd — APPROVED** [review]
🌟 🏅 🍰

**Pull request merged**